src/store/selectors.ts

```diff
@@ -296,14 +297,16 @@ export const selectTransferTargetError = (state: RootState) => {
     state.transfer.relayerFee === undefined &&
     // Acala offers relaying without a fee for qualified tokens
     state.transfer.targetChain !== CHAIN_ID_ACALA &&
-    state.transfer.targetChain !== CHAIN_ID_KARURA
+    state.transfer.targetChain !== CHAIN_ID_KARURA &&
+    state.transfer.targetChain !== CHAIN_ID_NEON
   ) {
     return "Invalid relayer fee.";
   }
   if (
     state.transfer.useRelayer &&
     (state.transfer.targetChain === CHAIN_ID_ACALA ||
-      state.transfer.targetChain === CHAIN_ID_KARURA) &&
+      state.transfer.targetChain === CHAIN_ID_KARURA ||
+      state.transfer.targetChain === CHAIN_ID_NEON) &&
     !state.transfer.acalaRelayerInfo.data?.shouldRelay
```

Comment:
Need to handle `state.transfer.neonRelayerInfo.data?.shouldRelay`

Reply:
Some think like this?
`  if (
    state.transfer.useRelayer &&
    (state.transfer.targetChain === CHAIN_ID_ACALA ||
      state.transfer.targetChain === CHAIN_ID_KARURA ||
      state.transfer.targetChain === CHAIN_ID_NEON) &&
    (!state.transfer.acalaRelayerInfo.data?.shouldRelay ||
      !state.transfer.neonRelayerInfo.data?.shouldRelay)
  ) {`

Reply:
Can we change it to:
```
    (state.transfer.useRelayer &&
      (state.transfer.targetChain === CHAIN_ID_ACALA ||
        state.transfer.targetChain === CHAIN_ID_KARURA) &&
      !state.transfer.acalaRelayerInfo.data?.shouldRelay) ||
    (state.transfer.targetChain === CHAIN_ID_NEON &&
      !state.transfer.neonRelayerInfo.data?.shouldRelay)
 ```

Reply:
Ok. I used your pattern.

Reply:
Re-check please.